script/ts/configs/AerodromeDexter/set-route.ts

```diff
@@ -0,0 +1,73 @@
+import { Command } from "commander";
+import { loadConfig } from "../../utils/config";
+import signers from "../../entities/signers";
+import { AerodromeDexter__factory } from "../../../../typechain";
+import { ethers } from "ethers";
+
+type Route = {
+  from: string;
+  to: string;
+  stable: boolean;
+  factory: string;
+}
+
+type SetRouteConfig = {
+  tokenIn: string;
+  tokenOut: string;
+  route: Route[];
+};
+
+async function main(chainId: number) {
+  const config = loadConfig(chainId);
+  const deployer = signers.deployer(chainId);
+  const dexter = AerodromeDexter__factory.connect(config.extension.dexter.aerodrome, deployer);
+
+  const params: Array<SetRouteConfig> = [
+    {
+      tokenIn: config.tokens.usdc,
+      tokenOut: config.tokens.wstEth,
+      route: [
+        {
+          from: config.tokens.usdc,
+          to: config.tokens.weth,
+          stable: false,
+          factory: config.vendors.aerodrome.factory
+        },
+        {
+          from: config.tokens.weth,
+          to: config.tokens.wstEth,
+          stable: false,
+          factory: config.vendors.aerodrome.factory
+        },
+      ]
+    },
+  ];
+
+  console.log("[cmds/AerodromeDexter] Setting path config...");
```

Comment:
Don't like how they use this preffix "[cmds/AerodromeDexter]" when we have console.group. Suggest to use in future console.group - it's avoid duplication of code. Can leave as it for now, just fyi
e.g.
```
console.group("[cmds/AerodromeDexter]")
...
console.log("Setting path config...");
...
console.groupEnd()
```